### PR TITLE
inserted hypen between  Add  and WindowsFeature

### PR DESCRIPTION
**Pull request by RAJU2529**
as per user report issue #1838 . so i  inserted  hypen  between  two words  Add and  WindowsFeature

### CODE REVIEW

**Comment by e0i** [comment]
@RAJU2529 Please change the target branch of the repository to `master`.

We know that online edits get targeted to `live` by default and we look into fixing that soon. Thank you.

**Comment by RAJU2529** [comment]
@e0i . changed  to master . thanks

**Comment by e0i** [comment]
@get-itips @dariomws Please approve the changes. Thank you.

File: docset/windows/adcsdeployment/Install-AdcsCertificationAuthority.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ You can import the cmdlet by running the following commands from Windows PowerSh
 - `Import-Module ServerManager`
 - `Add-WindowsFeature Adcs-Cert-Authority`
```
Comment (dariomws) [inline docset/windows/adcsdeployment/Install-AdcsCertificationAuthority.md]:
```suggestion
- `Install-WindowsFeature Adcs-Cert-Authority`
```

File: docset/windows/adcsdeployment/Install-AdcsCertificationAuthority.md [hunk 2]
```diff
@@ -62,7 +62,7 @@ You can import the cmdlet by running the following commands from Windows PowerSh
...
 
-To include the Certification Authority and Certificate Templates consoles in a CA installation, you must use the *IncludeManagementTools* parameter at the end of the `AddWindowsFeature Adcs-Cert-Authority` command.
+To include the Certification Authority and Certificate Templates consoles in a CA installation, you must use the *IncludeManagementTools* parameter at the end of the `Add-WindowsFeature Adcs-Cert-Authority` command.
```
Comment (dariomws) [inline docset/windows/adcsdeployment/Install-AdcsCertificationAuthority.md]:
```suggestion
To include the Certification Authority and Certificate Templates consoles in a CA installation, you must use the *IncludeManagementTools* parameter at the end of the `Install-WindowsFeature Adcs-Cert-Authority` command.
```

**Pull request merged**